src/integrationTest/java/uk/gov/hmcts/darts/audio/service/AudioTransformationServiceProcessAudioRequestTest.java

```diff
@@ -52,8 +53,9 @@ void setUp() {
 
     @ParameterizedTest
     @EnumSource(names = {"DOWNLOAD", "PLAYBACK"})
+    @Order(1)
```

Comment:
Is ordering the tests necessary? It's preferable not to order, as it suggest there is an ordering dependency between the tests, and ideally each test should be independent/standalone.